packages/scoped-custom-element-registry/src/scoped-custom-element-registry.ts

```diff
@@ -25,7 +25,7 @@ declare interface PolyfillWindow {
   };
 }
 
-const polyfillWindow = (window as unknown) as PolyfillWindow;
+const polyfillWindow = (globalThis as unknown) as PolyfillWindow;
```

Comment:
Calling attention to the actual non-formatting-related changes here.  This is one fix, to remove the raw reference to window in server environments.